README.md

```diff
@@ -6,13 +6,21 @@ The plan for the first week is to give you an overview of good things to know wh
 
 **Before you start, please make sure that you have all necessary software installed on your personal machine *before* we start the bootcamp. To help you along the way, have a look at [these instructions](SETUP-INSTALLATION.md).**
 
+For live questions of general interest to the "camp masters" and to keep in touch, please subscribe to the [PUAstro slack](https://join.slack.com/t/puastro/signup) workspace and to the [#usrp-bootcamp](https://puastro.slack.com/archives/C014EAUJFDL) channel.
+
```

Comment:
We should also add the zoom link.

Reply:
Please remove the links, we don't want to get spammed

Reply:
Done, I'll merge and squash.